Bottom: c362b995d882d1efbc8235b44469e18d908e9700
Top:    a0aaf7fc92e777350a93ef963a7603848faadf32
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 14:09:41 +0100

Ensure "Gathering" mode is set after painting the screen.



---

diff --git a/src/draw.c b/src/draw.c
index 8322879..f710567 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2711,4 +2711,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = True;
 }
